Small post promoting Ubuntu 24.04 LTS (#299)
readthedocs/website: content/posts/ubuntu-24-04.md

title: Ubuntu 24.04 LTS available

> date: 2024-06-19

Author: ```suggestion
date: 2024-06-21
```

description: The latest Ubuntu 24.04 LTS version is available to build documentation projects on Read the Docs.
category: Changelog
tags: builders, docker, images
authors: Manuel Kaufmann
status: published
image: /images/ubuntu-24-04.jpg
image_credit: Photo by <a href="https://unsplash.com/@6heinz3r?utm_content=creditCopyText&utm_medium=referral&utm_source=unsplash">Gabriel Heinzer</a> on <a href="https://unsplash.com/photos/green-and-black-digital-device-xbEVM6oJ1Fs?utm_content=creditCopyText&utm_medium=referral&utm_source=unsplash">Unsplash</a>

> We are happy to announce that we added support for the latest Ubuntu 24.04 LTS version released on April, 2024.

Reviewer: ```suggestion
We are happy to announce that we added support for the latest Ubuntu 24.04 LTS version released in April 2024.
```

This new image is available to build documentation projects by using the YAML config `build.os: "ubuntu-24.04"`.

We plan to update the `build.os: "latest"` alias after some weeks of testing this image.

> [Let us know](https://readthedocs.org/support/) any feedback you may have about this new Ubuntu 24.04 LTS image,

Reviewer: ```suggestion
[Let us know](https://docs.readthedocs.io/en/stable/support.html) any feedback you may have about this new Ubuntu 24.04 LTS image,
```
